engine/src/test/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActorSpec.scala

```diff
@@ -178,7 +179,7 @@ class EngineJobExecutionActorSpec extends CromwellTestkitSpec with Matchers with
 
       def ejeaInRunningState() = {
         val ejea = buildEJEA(restarting = true)
-        ejea.setState(stateName = RunningJob, stateData = EmptyPartialCompletionData)
+        ejea.setState(stateName = RunningJob, stateData = NoResponseData(null, Props.empty))
```

Comment:
FWIW I think we need to completely redux the EJEA spec. Happy enough to mark that as Tech Debt (tm) for now